### PR TITLE
Add arrow keys to view mode

### LINKED ISSUE #977: Add arrow keys to view mode
In normal mode `hjkl` are equivalent to the respective arrow keys for navigating around. In view mode, only `hjkl` are recognized. We should add them for consistency.

### PR DESCRIPTION
**Pull request by python128**
Fixes #977.

I have also added `S-keys` and `C-keys` to move page up/down and half-page up/down.

### CODE REVIEW

**Review by kirawi — APPROVED** [review]

**Comment by python128** [comment]
Should I update the same in the docs also?

**Comment by pickfire** [comment]
> Should I update the same in the docs also?

Yes

File: helix-term/src/keymap.rs [hunk 1]
```diff
@@ -591,24 +591,24 @@ impl Default for Keymaps {
...
-                "b" => page_up,
-                "f" => page_down,
-                "u" => half_page_up,
-                "d" => half_page_down,
+                "k" | "up" => scroll_up,
+                "j" | "down" => scroll_down,
+                "b" | "S-up" => page_up,
+                "f" | "S-down" => page_down,
+                "u" | "C-up" => half_page_up,
+                "d" | "C-down" => half_page_down,
```
Comment (pickfire) [inline helix-term/src/keymap.rs:611]: Where was ctrl-up based on? I don't see neovim or kakoune have this behavior.

Comment (python128) [reply helix-term/src/keymap.rs:611]: I thought it would be a nice feature to have, especially if you want to scroll down only some amount of lines..

**Comment by python128** [comment]
> > Should I update the same in the docs also?
> 
> Yes

Where to do so? I am not sure.

File: helix-term/src/keymap.rs (hunk 1 above)
Comment (archseer) [reply helix-term/src/keymap.rs:611]: Drop these `C-up`/`C-down` mappings, I'd prefer users add these using their local config.

**Comment by python128** [comment]
> Where to do so? I am not sure.

NP, done.

File: helix-term/src/keymap.rs [hunk 2]
```diff
@@ -591,10 +591,10 @@ impl Default for Keymaps {
...
                 "b" => align_view_bottom,
                 "m" => align_view_middle,
-                "k" => scroll_up,
-                "j" => scroll_down,
-                "b" => page_up,
-                "f" => page_down,
+                "k" | "up" => scroll_up,
+                "j" | "down" => scroll_down,
+                "b" | "S-up" => page_up,
+                "f" | "S-down" => page_down,
```
Comment (archseer) [inline helix-term/src/keymap.rs:597]: We're changing these too https://github.com/helix-editor/helix/pull/1040

Can you only keep the aliases for `k`/`j`?

Comment (python128) [reply helix-term/src/keymap.rs]: Done 👍

Comment (python128) [reply helix-term/src/keymap.rs]: Edited the `kepmap.md` also

**Comment by archseer** [comment]
Can you rebase once more? Then I'll finally merge :)

**Comment by python128** [comment]
There is some conflict..

**Comment by archseer** [comment]
Yeah, you didn't rebase on latest `origin/master` it seems

**Pull request merged**